packages/smart-contract/test/SmartWalletManager.test.ts

```diff
@@ -15,6 +15,7 @@ import { Dai } from '../typechain/Dai';
 chai.use(solidity);
 const { expect } = chai;
 const ZERO = '0x0000000000000000000000000000000000000000';
+let smartWalletAddress = ZERO;
```

Comment:
you can use constant from ethers https://github.com/ethers-io/ethers.js/blob/master/src.ts/constants.ts#L3